dev-docs/source/GettingStarted/GettingStarted.rst

```diff
@@ -15,17 +15,24 @@ Getting Started
    GettingStartedCondaOSX
    GettingStartedNoneConda
 
-The Choice
-##########
+Dependencies
+############
 
-With the addition of Conda to our workflow we at present have two different methods of Getting Started with mantid
-development. Therefore you can choose which method to setup with (We recommend Conda as it will be the only supported
+The dependencies required to build Mantid will vary depending on your platform and the method you choose to setup with. At the very least you will need:
+
+* `Git <https://git-scm.com/>`_
```

Comment:
This is more of a suggestion, so feel free to push back! I'm not sure if its helpful to list git as a dependency here, as its already listed in the "Getting started" instructions for each of the OS's